src/overlays/actors/ovl_En_Test/z_en_test.c

```diff
@@ -257,7 +257,7 @@ void EnTest_Draw(Actor* thisx, PlayState* play) {
         sp2C = 29;
     }
 
-    if ((this->unk_20A == 15) || (this->unk_20A == 14)) {
+    if ((this->surfaceMaterial == SURFACE_MATERIAL_MAX) || (this->surfaceMaterial == SURFACE_MATERIAL_SNOW)) {
```

Comment:
This seems odd that it's checking for `SURFACE_MATERIAL_MAX`, that shouldn't be a valid material?

Reply:
It shouldn't be a valid material, but `SurfaceType_GetMaterial` doesn't do any kind of check to prevent returning that value, so there may be some very weird edge case where a surface is buggy and has a 0xF here.
Should we tag this as a `@bug`?

Reply:
So a surfaceMaterial of 0xF would read into the padding since a lot of the data would be 15 long? So it would be interpreted as dirt in a few ways.
And we're sure it's 15, not 16 materials? I suppose the `surfaceMaterial >= ARRAY_COUNT(sSurfaceSfxOffsets)` in `SurfaceType_GetSfxOffset` indicates that it's 15.
Might be worth a bug tag yeah

Reply:
Yeah that specific check tell us the max is 15.

Not sure how to word this bug tag
```
//! @bug Checks for non-existent SURFACE_MATERIAL_MAX material
```

Reply:
Yeah, that's fine I think